apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -59,25 +59,34 @@ class SectionAssessments extends Component {
             scriptId={scriptId}
             onChange={this.onChangeScript}
           />
+          {!isLoading &&
+            <div>
+              <div style={{...h3Style, ...styles.header}}>
+                {i18n.selectAssessment()}
+              </div>
+              <AssessmentSelector
+                assessmentList={assessmentList}
+                assessmentId={assessmentId}
+                onChange={this.props.setAssessmentId}
+              />
+            </div>
+          }
         </div>
-        <div>
-          <div style={{...h3Style, ...styles.header}}>
-            {i18n.selectAssessment()}
+        {!isLoading &&
```

Comment:
Why not combine these into 1 `!isLoading` block?

Reply:
It does look a little strange because they seem right next to each other. Styling requires the selectors to be together in one div, which means a single selector and the tables can't actually be in one element :( I think it'll be more obvious once we add more controls to the selector area.